db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <preConditions>
+        <dbms type="mysql"/>
+    </preConditions>
+    
+    <changeSet id="0" author="aktsay">
+        <tagDatabase tag="db_init"/>
+    </changeSet>
+
+    <include file="v-0.0.1/db.changelog-v-0.0.1.xml" relativeToChangelogFile="true"/>
```

Comment:
Please rename directories as we have discussed

Reply:
Fixed